templates/appliance/governance.html

```diff
@@ -0,0 +1,132 @@
+      </ul>
+      <p>In the docs section, you will find access to the image builder. A tool where anyone can build appliance images out of whichever snaps they please to test on their own. Any images this way are not guaranteed to work and are also considered experimental.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-8">
+      Hardware compatibility
+      Ubuntu Core is guaranteed to work on any Ubuntu certified hardware. This includes Raspberry Pi’s and the Intel NUC. Certification means that these boards exist in Canonical’s certification and testing lab and every time an update, bug fix or security patch is rolled out to the OS it is tested on these boards first. This is to ensure no update breaks a user’s appliance. Before a project can become an official Ubuntu appliance, it must be tested to work on a Raspberry Pi or Intel NUC.
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-8">
+      <h2>Decision-making bodies</h2>
+      <p>In order to have your project published as an official Ubuntu appliance and featured in the portfolio, you must get in touch with the right people at Canonical. The easiest way to do this is through the snapcraft forum, where Canonical engineers can help you package your software as a snap and test your images. It is then a simple case of creating a post in the snapcraft forum or making a topic in the Ubuntu Discourse. After that, the decision rests with the Ubuntu Community Council. The primary community (i.e., non-technical) governance body for the Ubuntu project. Members are appointed by Mark Shuttleworth and approved by a vote by the entire Ubuntu membership.</p>
+      <h3>Ubuntu Community Council</h3>
+      <p>The social structures and community processes of Ubuntu are supervised by the Ubuntu Community Council, which manages nominations and elections for Ubuntu boards and councils. The council is also responsible for the Code of Conduct and tasked with ensuring that community members and leaders live up to the standard it sets.</p>
+      <p>The Council is ultimately responsible for dispute resolution, should it be required. For example, in the past, we have helped to resolve conflicts in LoCo teams and in the Ubuntu forums - both very important parts of the community that have their own leadership structures carrying authority delegated by the Community Council.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-8">
+      <h2>Commercial terms</h2>
+      <p>In order to use an Ubuntu appliance image in a commercial product, you must <a href="">contact Canonical</a> so that we can talk to the developer about the use of their work and Ubuntu Core in your product. You must see the appliances and Canonical’s licensing terms before using the image in any commercial product.</p>
```

Comment:
link's to no where? I think the comments says there is a page?  For now you could link to /core/contact-us?

Reply:
It does now